components/aws-authservice/README.md

```diff
@@ -0,0 +1,50 @@
+# AWS AuthService
+
+## Overview
+AWS AuthService is an HTTP Server that handles the logging out of an Authenticated user who was connected to Kubeflow using AWS Cognito and Amazon ALB.
+
+## Design
+An HTTP Server that listens for a users logout request that then follows the two steps necessary to logout an Authenticated Cognito + ALB user. These being expiring any ALB Cookies and then hitting the Cognito Logout Endpoint. Official [Documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/listener-authenticate-users.html#authentication-logout) lists these steps as required for secure logout.
+
+## Manifests
+To install AWS AuthService apply them to your EKS Cluster. The manifests can be found in [awsconfigs](../../awsconfigs/common/aws-authservice/base/).
+
+```
+kubectl apply -k ../../awsconfigs/common/aws-authservice/base/
+```
+
+These are configurable environment variables used by AWS AuthService in [params.env](../../awsconfigs/common/aws-authservice/base/params.env)
+
+`LOGOUT_URL` [REQUIRED]: The Cognito URL that will be redirected to on Logout.
+
+## Build and Test
+If you wish to make custom changes to AWS AuthService you can modify [main.go](main.go)
+
+The image can be built using Docker.
+- make build
+
+Tag and push the image to ECR or any Container Image Library i.e(DockerHub)
+- make tag IMAGE_TAG=<>
```

Comment:
please change it to `IMAGE_URI` instead of just tag because each person will have different registry. the format for image consist of registry, repository and tag like`REGISTRY/REPO_NAME:TAG` e.g. `surajkota/auth-service:1.1.0`